sentry-android-core/src/test/java/io/sentry/android/core/internal/util/PrivilegeEscalationViaContentProviderCheckerTest.kt

```diff
@@ -0,0 +1,105 @@
+package io.sentry.android.core.internal.util
+
+import android.content.ContentProvider
+import android.content.Context
+import android.os.Build
+import io.sentry.android.core.BuildInfoProvider
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verifyNoInteractions
+import org.mockito.kotlin.whenever
+import kotlin.test.Test
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+class PrivilegeEscalationViaContentProviderCheckerTest {
+
+    private class Fixture {
+        val buildInfoProvider = mock<BuildInfoProvider>()
+
+        fun getSut(
+            sdkVersion: Int = Build.VERSION_CODES.O
+        ): PrivilegeEscalationViaContentProviderChecker {
+            whenever(buildInfoProvider.sdkInfoVersion).thenReturn(sdkVersion)
+
+            return PrivilegeEscalationViaContentProviderChecker(buildInfoProvider)
+        }
+    }
+
+    private val fixture = Fixture()
+
+    @Test
+    fun `When sdk version is less than vulnerable versions, security check is not performed`() {
```

Comment:
Minor stuff, it appears the Sentry team [ is utilising](https://github.com/getsentry/sentry-java/blob/1e1ab7f5a4a1586225c60ae343015e012649ef26/sentry-android-core/src/test/java/io/sentry/android/core/internal/util/SentryFrameMetricsCollectorTest.kt#L91) lowercase symbols in their test methods.

Reply:
I think they use both upper and lowercase then because I copy-pasted-and-modified their existing `RootCheckterTest.kt` file for this; https://github.com/getsentry/sentry-java/blob/6.11.0/sentry-android-core/src/test/java/io/sentry/android/core/internal/util/RootCheckerTest.kt#L48

All tests in that file are in the format "When ..., ...."